index.html

```diff
@@ -364,6 +365,11 @@ <h2><dfn>disconnect()</dfn> method</h2>
         <a>relevant global object</a>, and also empty <a>context object</a>'s
         <a>observer buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>takeRecords()</dfn> method</h2>
+        <p>The <a>takeRecords()</a> method returns a copy of the <a>observer
+          buffer</a> and empties the observer buffer.</p>
```

Comment:
I think this needs to be the context object's observer buffer.

Reply:
Done